docs/reference/nodes/workflow.md

```diff
@@ -0,0 +1,44 @@
+# Workflow Nodes
+
+
+## Input & Output Nodes
+*Input* and *Output* nodes are used to define the required inputs and outputs of a workflow to be executed externally. Setting these nodes is essential to be able to nest workflows or executing the workflow in [batch mode](../../getting_started/basics/batch.md). The custom plugs from these nodes, which should be of type `through`, will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa. 

```

Comment:
```suggestion
*Input* and *Output* nodes are used to define the required inputs and outputs of a workflow to be executed externally. Setting these nodes is essential to be able to nest workflows or executing the workflow in [batch mode](../../getting_started/basics/batch.md). The custom plugs from these nodes, which should be of type `through`, will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa. These nodes will also be crucial to pass external inputs or retreiving results when executing Shift workflows in Batch mode.

```

Reply:
The paragraph already mentions this it is used for batch execution. Should we be more specific for what having this nodes means for interactive mode vs batch mode?

Reply:
my bad!